article/performance-comparisons.tex

```diff
@@ -112,3 +112,39 @@ \subsubsection{Hyperparameter experiment}
     \caption{Recall$@$100k on NGCF PAS with different input values}
     \label{fig:recall-ngcfpas-weights}
 \end{figure}
+
+\subsubsection{One convolution experiment}
+An experiment was conducted to see the influence of only using one convolution in the different methods.
+The results can be seen in \autoref{tab:one-convolution}.
+When comparing the results with the experiment in \autoref{tab:results-with-many-methods} that used 3 convolutions for all methods except PUP, it can be seen that NGCF and LightGCN performs a bit worse for Recall@50 and NDCG@50.
```

Comment:
```suggestion
When comparing the results with the previous experiment in \autoref{tab:results-with-many-methods} that used 3 convolutions for all methods except PUP, it can be seen that NGCF and LightGCN have a worse performance when evaluating them with Recall@50 and NDCG@50.
```